tarpc/src/transport/mod.rs

```diff
@@ -0,0 +1,84 @@
+use std::io::{self, Read, Write};
+use std::time::Duration;
+
+/// A factory for creating a listener on a given address.
```

Comment:
Yeah just something like "for TCP an address might be an IPV4 address, for unix sockets it is just a file name" or something along those lines.